schemas/Clinical/index.js

```diff
@@ -312,16 +332,27 @@ const resolvers = {
 
       const { filename, createReadStream } = await registrationFile;
       const fileStream = createReadStream();
-      const response = await clinicalService.uploadRegistrationData(
-        shortName,
-        filename,
-        fileStream,
-        Authorization,
-      );
 
-      // Success data is inside the key "registration", error data is in the root level
-      const data = response.successful ? response.registration : response;
-      return convertRegistrationDataToGql(data);
+      try {
+        const response = await clinicalService.uploadRegistrationData(
+          shortName,
+          filename,
+          fileStream,
+          Authorization,
+        );
+
+        // Success data is inside the key "registration", error data is in the root level
+        const data = { ...response.registration, errors: response.errors };
+        return convertRegistrationDataToGql(data);
+      } catch (e) {
+        // errors that don't go into error table
+        if (e.code) {
+          return { error: e.msg, code: ERROR_CODES[e.code] };
+        } else {
+          // catch all error
+          console.error('other error', e);
```

Comment:
there's a `winston` logger that can be used, import from `utils/logger.js` :) it'll comes with timestamp and log level too

Reply:
use winston logger

Reply:
thank you!